src/catalog/inventory-orders-shipment.md

```diff
@@ -28,7 +28,7 @@ _Items to Refund with Return to Stock Selected_
 
 ### Cancel unshipped orders
 
-If an order has not been shipped and is canceled (in full or partial), Inventory Management automatically returns the product stock to the salable quantity. Until invoice and shipping, purchased products are reserved against the salable quantity, not deducted from the actual quantity. Invoicing and shipping converts the reservation to an inventory deduction.
+If an order has not been shipped and is canceled (in full or partial), Inventory Management automatically returns the product stock to the salable quantity. Until invoice and shipping, purchased products are reserved against the salable quantity, not deducted from the actual quantity. Invoicing and shipping convert the reservation to an inventory deduction.
```

Comment:
I believe that the original writer intended to use the singular verb because they considered "Invoicing and Shipping" as a single entity and not two separate things. However, I do think that the construction here is confusing. I am suggesting the following change to resolve the issue without implying that more than one conversion is taking place:

```suggestion
If an order has not been shipped and is canceled (in full or partial), Inventory Management automatically returns the product stock to the salable quantity. Until invoice and shipping, purchased products are reserved against the salable quantity, not deducted from the actual quantity. At the point of invoicing and shipping the order, the system converts the reservation to an inventory deduction.
```